pkgs/stdenv/darwin/default.nix

```diff
@@ -7,15 +7,17 @@
 # Allow passing in bootstrap files directly so we can test the stdenv bootstrap process when changing the bootstrap tools
 , bootstrapFiles ? let
   fetch = { file, sha256, executable ? true }: import <nix/fetchurl.nix> {
-    url = "http://tarballs.nixos.org/stdenv-darwin/x86_64/5ab5783e4f46c373c6de84deac9ad59b608bb2e6/${file}";
+    # These were copied from https://hydra.nixos.org/build/132673674
+    # TODO: replace with nixos tarballs mirror
+    url = "https://s3.ap-northeast-1.amazonaws.com/nix-misc.cons.org.nz/stdenv-darwin/x64_64/3b85d7d54f083f46cf06eddcd7694a9078ad13d3/${file}";
```

Comment:
What changed since the last build that's needed for bootstrapping? I would update this a final time once everything has stabilised instead if it's not required now.

Reply:
The difference in the build is that the currently used bootstrap tools were built against the proxy libraries, and new bootstrap tools would be built against the tbd stubs. But this doesn't actually affect anything in the resulting binaries, as far as I can tell, so there is no need for this => removed.